src/game/eop.js

```diff
@@ -1,8 +1,8 @@
-import { Game, INVALID_MOVE, PlayerView } from 'boardgame.io/core';
+import { TurnOrder, INVALID_MOVE, PlayerView } from 'boardgame.io/core';
```

Comment:
Idea (tell me if this is too much work to do):
Could we move the moves etc. to separate files?
Then we have a nice separation, such that we could write tests for the moves. Because in principle, they should easily be testable.

Reply:
This should be really easy to implement and I think it will make it a lot neater

Reply:
moves are in moves.js, this file was already quite big so all the other functions are in utils.js

I wasn't sure what to call this second file, let me know if you have a better suggestion